include/file.iol

```diff
@@ -77,6 +77,20 @@ type ListResponse:void {
 interface FileInterface {
 RequestResponse:
 	/**!
+	 * Constructs an absolute path to the target file or directory. 
+	 * Can be used to construct an absolute path for new files that does not exist yet.
+	 * Throws a InvalidPathException fault if input is a relative path is not system recognized path.
+	 */
+	toAbsolutePath( string )( string ) throws JavaExceptionType( InvalidPathExceptionType ),
```

Comment:
Huh? How does this work?

Should probably be: `throws InvalidPathException( JavaExceptionType )`

Same below.